backend/src/index.ts

```diff
@@ -14,11 +17,13 @@ app.use(cors({
     credentials:true
 }))
 
-app.get("/health",(req,res)=>{
-    res.status(200).json({
```

Comment:
We can use http.ts variables instead of http code, so that it can be human readable